### DISCUSSION TITLE
How to use pytorch3d with an amd gpu?

### DISCUSSION THREAD

**Discussion by joaodalino** [facebookresearch/pytorch3d#750] [Q&A]
Hey, i'm newbie in 3d meshing. I was trying to install pytorch3d without having a nvidia gpu, but when i proceed to the installation, my pc just crash. It's possible to use it with an amd gpu?

**Comment by bottler** [comment]
The compiled parts of PyTorch3D don't include ROCm support. You should be able to build a CPU-only version of PyTorch3D, and you should have no problem installing a prebuilt PyTorch3D if you are on Linux. But any of the compiled parts of PyTorch3D will only work with CPU tensors, and I would not be surprised if they crash if given GPU tensors.

Can you say more about your install and what went wrong?
